plugin.yaml

```diff
@@ -173,6 +173,17 @@ providers:
         required: false
         renderingOptions:
           groupName: Kerberos
+      - name: cleanescaping
+        title: Clean Escaping
+        description: "Cleans unnecessarily Escaped characters on commands"
+        type: Select
```

Comment:
we should use a boolean type, not select here

Reply:
I did the changes that you ask me.